src/asset_folder_vsingester.py

```diff
@@ -39,8 +40,25 @@ def file_has_any_extension(filename, extensionlist):
 
     return False
 
+
+def test_portal_connection(filepath, cfg):
+    from asset_folder_importer.pluto.assetfolder import AssetFolderLocator, ProjectNotFound
+    tester = AssetFolderLocator(scheme=cfg.value('pluto_scheme',default="http"), host=cfg.value('pluto_host'), port=cfg.value('pluto_port'),
+                           user=cfg.value('vs_user'), passwd=cfg.value('vs_password'))
+    try:
+        tester.find_assetfolder(filepath)
+    except ProjectNotFound:
+        pass
+    return None
+
+
 #This function is the main program, but is contained here to make it easier to catch exceptions
 def innerMainFunc(cfg,db,limit, keeplist):
+    try:
+        test_portal_connection('/start/up/test.mxf', cfg)
+    except SweeperHTTPError:
+        logging.critical("Error accessing Portal. Bailing out.")
+        raise PortalHTTPError
```

Comment:
do we need to raise a second, different exception here? could't we just raise SweeperHttpError at line 126 and then catch SweeperHttpError in the calling function, log the critical error and the database there?

Reply:
You said I needed to use a new custom exception here so I did. If you want I can rewrite the code without it.

Reply:
I have attempted to write it without the extra exception.